### PR TITLE
feat(docs) Adds guide on how to use personal access tokens

### PR DESCRIPTION
**Pull request by pedro93**
## Checklist
- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

### CODE REVIEW

**Review by szalai1 — COMMENTED** [review]
@pedro93 static assets should go to https://github.com/datahub-project/static-assets
AFAIK

**Review by aditya-radhakrishnan — APPROVED** [review]
LGTM! @jjoyce0510 can you confirm whether static assets should stay in the imgs directory or be moved to the other repo?

File: docs/authentication/personal-access-tokens.md [hunk 1]
```diff
@@ -1,26 +1,50 @@
...
+
+:::
+
+## Creating Personal Access Tokens
+
+Once in the Manage Access Tokens Settings Tab:
+
+1. Click `Generate new token` where a form should appear.
+
+![](../imgs/pats/token-creation-form.png)
```
Comment (maggiehays) [inline docs/authentication/personal-access-tokens.md:35]: Couple of nit-picks for consistency with other guides & improved readability:

* images should be stored in https://github.com/datahub-project/static-assets
*  please use HTML to embed images so we can size down (70%) & center-align:

```
<p align="center">
  <img width="70%"  src="https://raw.githubusercontent.com/datahub-project/static-assets/master/imgs/token-creation-form.png"/>
</p>
```

File: docs/authentication/personal-access-tokens.md [hunk 2]
```diff
@@ -1,26 +1,50 @@
...
+
+2. Fill out the information as needed and click `Create`.
+
+![](../imgs/pats/filled-out-form.png)
```
Comment (maggiehays) [inline docs/authentication/personal-access-tokens.md:39]: Same here - move to `/static-assets` repo & embed with HTML

File: docs/authentication/personal-access-tokens.md [hunk 3]
```diff
@@ -1,26 +1,50 @@
...
+
+3. Save the token text somewhere secure! This is what will be used later on!
+
+![](../imgs/pats/token-info.png)
```
Comment (maggiehays) [inline docs/authentication/personal-access-tokens.md:43]: Same here - move to `/static-assets` repo & embed with HTML

File: docs/authentication/personal-access-tokens.md [hunk 4]
```diff
@@ -45,3 +73,38 @@ Without an access token, making programmatic requests will result in a 401 resul
...
+- Learn more about how this feature is by DataHub [Authentication Metadata Service](introducing-metadata-service-authentication.md).
+- Check out our [Authorization Policies](../authorization/policies.md) to see what permissions can be programatically used.
+
+### GraphQL
+
+ - Have a look at [Token Management in GraphQL](../api/graphql/token-management.md) to learn how to manage tokens programatically!
+ 
+## FAQ and Troubleshooting
+
+ <!-- Use the following format:
```
Comment (maggiehays) [inline docs/authentication/personal-access-tokens.md:88]: Let's remove commented out details from the template unless they capture doc-specifc details

**Review by maggiehays — CHANGES_REQUESTED** [review]
@pedro93 This is an AWESOME improvement to the existing guide - much easier to follow and approable. Left a couple of comments, but otherwise this is 🔥

**Comment by pedro93** [comment]
Comments addressed, this PR depends on https://github.com/datahub-project/static-assets/pull/8/files being merged.

**Review by maggiehays — APPROVED** [review]

**Pull request merged**